include/bluetooth/audio/cap.h

```diff
@@ -1,97 +0,0 @@
-/*
```

Comment:
This commit ("Add ASCS Metadata verification") removes the CAP header file introduced in the previous commit.  This seems like a mistake.
A smaller version of the header file is added back again in commit "Add CAP acceptor set member support".